components/utils/react-holder.js

```diff
@@ -1,24 +1,33 @@
+/**
+ * Utils - React Holder
+ *
+ * @see https://github.com/imsky/holder#placeholder-options for info on more props and themes
+ */
 import React, { useEffect, useRef } from 'react'
 import ReactDOM from 'react-dom'
 import { string, bool, object } from 'prop-types'
-// import Holder from 'holderjs'
 
-const ReactHolderJs = ({ src, width, height, usePlaceholder }) => {
+export const ReactHolderJs = ({ src, width, height, usePlaceholder }) => {
```

Comment:
`WARNING in ./components/molecules/carousel/__tests__/stories.js 31:29-32
"export 'default' (imported as 'Img') was not found in '../../../utils/react-holder'`